api/observability/v1/filter_types.go

```diff
@@ -64,6 +64,15 @@ type FilterSpec struct {
 
 	// Type of filter.
 	//
+	// Possible filter types are:
+	//
+	// 1. detectMultilineException
+	// 2. drop
+	// 3. kubeAPIAudit
+	// 4. openshiftLabels
+	// 5. parse
+	// 6. prune
```

Comment:
Maybe make sense add short description of filter ability:
drop - allow to drop unwanted log records 
kubeAPIAudit - allow to remove unwanted events and reduce event size to create a manageable audit trail 
prune - allow to prunes log records by removing low value fields such as pod annotations
and etc